pkg/rtc/mediatrackreceiver.go

```diff
@@ -97,32 +94,26 @@ type MediaTrackReceiverParams struct {
 }
 
 type MediaTrackReceiver struct {
-	params      MediaTrackReceiverParams
-	muted       atomic.Bool
-	simulcasted atomic.Bool
+	params MediaTrackReceiverParams
 
 	lock            sync.RWMutex
 	receivers       []*simulcastReceiver
-	receiversShadow []*simulcastReceiver
```

Comment:
This was unnecessary copy of the original slice. Did not serve anything useful as far as I could see.